### ISSUE TITLE
Portfolio Design & Content Rationale

### ISSUE THREAD

**Issue by alexhealy** [alexhealy/portfolio#9]
| | 0 points | 1 points | 2 points | 3 points |
| --- | --- | --- | --- | --- |
| **Pages & images** | Barely started | Missing lots of pages & images | Has most of the pages, some placeholder images | All pages exist and are well done with complete imagery |
| **Personality & design** | Bland, boring, just another graphic designer | Fairly generic, visual design could be pushed more | Quality layouts and unique, personable content & design | Unique and recognizable as you, with engaging, personable content & design |
| **Image quality** | Only placeholder images | Some portfolio piece images | Lots of images but low quality | High quality portfolio images — and lots of them |
| **Accessibility** | No considerations | Added some alt attributes, nothing else | Just the basics: alt attributes, roles | Accessibility is well considered and tested |
| **Text content** | Fake content only | Some real content, some fake | All real content but with lots of errors | Real content, well written, no grammar or spelling errors |
| **Responsiveness** | Not responsive | Works on some screens | Responsive but with lots of awkwardness | Looks great on all screen sizes |
| **Code quality** | Barely started | Indentation is barely existent, lots of validation errors, very poor semantics | Decent indentation, just a couple validation errors, decent semantics | Well indented, fully valid, good semantics |

Overall, I believe that I should receive an A on my portfolio website.

I was able to design and create a responsive portfolio website that showcases my brand and portfolio identity through its use of colours and content design. I kept my layout simple and clean, keeping the main focus on my project pieces. I feel that I have successfully used the knowledge learned throughout the past web development courses in order to create this final portfolio website, meeting all elements needed, such as the use of HTML, CSS & Java, as well as Jekyll, Markdown and YAML.

I believe that this portfolio will be utilized in the future, with its clean and organized code that has the ability to be changed and altered with its easy accessibility. 

An area of my portfolio website that could be improved includes the loading time of my pages, due to my video sizes. Another area of improvement could be the ability to view the "still" images larger when clicked or hovered on.

I did not run into too many major problems however one major problem was implementing my videos as a banner and playing them in general. This included converting the mov. format into the proper mp4. format, however the video still seemed to be having issues. The videos seemed to work the first time playing, but throughout opening the different pages, the videos would slowly stop automatically playing. This was solved by re exporting the videos from After Effects into Media Encoder which seemed to fix the issues automatically playing the video when a page was opened.

In conclusion, I believe I have implemented and used all code necessary in order to complete a responsive and creative portfolio website that properly brands my identity and showcases my work. I think it shows my ability to create content pieces for various social media platforms. Each section has the capability to be changed and altered easily for future reference and usage as well. 

@thomasjbradley
